doc/api/debugger.md

```diff
@@ -193,4 +193,13 @@ e.g. `--inspect=9222` will accept DevTools connections on port 9222.
 To break on the first line of the application code, provide the `--debug-brk`
 flag in addition to `--inspect`.
 
+```txt
+$ ./node --inspect index.js
+Debugger listening on port 9229.
+Warning: This is an experimental feature and could change at any time.
+To start debugging, open the following URL in Chrome:
+    chrome-devtools://devtools/remote/serve_file/@60cd6e859b9f557d2312f5bf532f6a
```

Comment:
Nit: maybe it's better to not split this line as it is a URL. I'm not sure though because if it isn't split scrolling is required on small screens.